cypress/page/shopping-cart.page.ts

```diff
@@ -0,0 +1,12 @@
+class ShoppingCartPage {
+    private shoppingCart: string;
+
+    constructor() {
+        this.shoppingCart = ".cart_navigation span";
+    }
+    public Cart(): void {
```

Comment:
since we are clicking in a button called proceed to checkout, you can call it like ProceedToCheckout o GoToCheckout

Reply:
change to GoToCheckout